HoI4Mod/Rome_Strikes_Back/common/national_focus/UTA_NF.txt

```diff
@@ -279,7 +258,7 @@ focus_tree = {
 			is_in_faction_with = CUB
 			CUB = { has_government = democratic }
 		}
-		ai_will_do = { factor = 90 }
+		ai_will_do = { factor = 70 }
 		completion_reward = {
 			declare_war_on = {

```

Comment:
It's generally better practice to give a wargoal, rather than directly declare the war - both to avoid declaring on allies in case of weirdness, and to allow for flexibility in case a war isn't the best choice immediately.

Also, focuses that give wargoals or war decs, should use `will_lead_to_war_with = <tag>` (this alerts the AI and player accordingly)